src/trie/merkle_tree.rs

```diff
@@ -112,6 +118,39 @@ fn test_shared_path_encode_decode() {
     let decoded = Path::decode(&mut &encoded[..]).unwrap();
     assert_eq!(path, decoded);
 }
+/// A node used in proof generated by the trie.
+///
+/// See pathfinders merkle-tree crate for more information.
```

Comment:
can you either add the link to the doc or add a small summary plz

Reply:
Done